webpack.config.js

```diff
@@ -235,7 +235,7 @@ export default {
           return {name, version, licenseName, body: formatLicenseText(licenseText)};
         });
 
-        const modules = [...goModules, ...jsModules].sort((a, b) => a.name.localeCompare(b.name));
+        const modules = [...goModules, ...jsModules].sort((a, b) => (a.name || '').localeCompare(b.name || ''));
```

Comment:
It looks strange, why ` || ''`

Reply:
With pnpm and the updates to the license checker plugin, it was catching a few modules that had an empty name, so I added these as protection. My PR to the webpack plugin also has similar protections, so I can remove this if you don't think it's necessary.

Reply:
I have no idea whether it is necessary since I don't know the details. It just looks strange to me. IMO it needs some comments to explain why the fallback it needed, or don't use the fallback and fix the root problems in related packages.

Reply:
```suggestion
        const modules = [...goModules, ...jsModules].sort((a, b) => a.name.localeCompare(b.name));
```

Fair enough. It's handled upstream already, and this is just an artifact from before I fixed it upstream so I'll remove it just to keep things clean :)